.github/workflows/sugar_unit_test.yaml

```diff
@@ -34,12 +34,24 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: subosito/flutter-action@v2.18.0
+      - name: Install Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20"
+      - name: Install Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: "adopt"
+          java-version: "11.0.26+4"
+      - name: Set up Maven
+        uses: stCarolas/setup-maven@v5
```

Comment:
I don't think we need a separate Maven action as set-up Java should already provide it.

Reply:
It sadly does not.